hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/s3/multipart/S3MultipartUploadCompleteResponse.java

```diff
@@ -147,6 +148,7 @@ protected List<OmKeyInfo> getPartsUnusedList() {
     return partsUnusedList;
   }
 
+  @CheckForNull
```

Comment:
the public method, `getOmBucketInfo()` is unnecessary with this change.  Now it is no longer used from the `S3MultipartUploadCompleteResponseWithFSO`, as we just use the `bucketID`.  The method can be removed.

Also please check if `S3MultipartUploadCompleteResponse.getPartsUnusedList `method is unused.  If so, it to can be removed.
  https://github.com/apache/ozone/blob/037492f839903c67709d44bd5a4ea9ca9ea093a9/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/s3/multipart/S3MultipartUploadCompleteResponse.java#L146

Reply:
`getOmBucketInfo()`, `getPartsUnusedList()` and `getMultipartKey()` are unused. I've done with this commit: https://github.com/apache/ozone/pull/4048/commits/955ed39197104e174a7971ff609e7b609d6c162d

Reply:
Sorry, I revised the commit for cleanup: https://github.com/apache/ozone/pull/4048/commits/ea2152696d9f928a57987ee57a8b1b7570c1894f; checked at my local and now checking with the CI...

Reply:
looks good.  Thanks for the cleanup.